site/gatsby-site/src/components/submissions/SubmissionWizard.js

```diff
@@ -15,15 +15,25 @@ const SubmissionWizard = ({ submitForm, initialValues }) => {
 
   const [steps, setSteps] = useState([]);
 
+  const [submissionFailed, setSubmissionFailed] = useState(false);
+
   const [parsingNews, setParsingNews] = useState(false);
 
   const stepsRef = useRef(null);
 
   const handleNextStep = async (newData, final = false) => {
+    setSubmissionFailed(false);
     setData((prev) => ({ ...prev, ...newData }));
 
     if (final) {
-      await submitForm({ ...data, ...newData });
+      try {
+        await submitForm({ ...data, ...newData });
+      } catch (error) {
+        setTimeout(() => {
```

Comment:
This will result in a race condition. Why is it necessary? Does `0` work? It is a bit better.

Reply:
0 works!